lib/constants/programming-language.ts

```diff
@@ -0,0 +1,14 @@
+export enum ProgrammingLanguage {
```

Comment:
Please do this in a seperate PR, as the platform changes are a lot to review.

Reply:
Should I create separate issue in this case?